node/src/chain_spec.rs

```diff
@@ -245,19 +254,31 @@ pub fn local_testnet_config() -> ChainSpec {
 						AccountId::from(hex!(
 							"caeedb2ddad0aca6d587dd24422ab8f6281a5b2495eb5d30265294cb29238567"
 						)),
-						AuraId::from_slice(&hex!(
-							"caeedb2ddad0aca6d587dd24422ab8f6281a5b2495eb5d30265294cb29238567"
-						))
-						.unwrap(),
+						(
+							AuraId::from_slice(&hex!(
+								"caeedb2ddad0aca6d587dd24422ab8f6281a5b2495eb5d30265294cb29238567"
+							))
+							.unwrap(),
+							BeefyId::from_slice(&hex!(
```

Comment:
Does it work? BeefyId should take [u8; 33]

Reply:
I tested it somehow :-) I believe it does.